dead-simple-k8s/hostIP.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+sleep 10
+curl -s "${K8S_API:-localhost:8001}/api/v1/namespaces/$NS/pods/$POD_NAME" | jq '.status.hostIP' | sed 's/"//g'
```

Comment:
Doing it as a wrapper looks nice.  I agree that's probably better.

jq is a tall order but how do you suggest interacting with the kubernetes API?  busybox probably doesn't even have curl...